modules/prebidServerBidAdapter/index.js

```diff
@@ -484,6 +485,18 @@ export function PrebidServer() {
               adapterMetrics
             });
           }
+          // pbs analytics event
+          if (emitSeatNonBid || emitAtag) {
+            const data = {
+              seatnonbid: emitSeatNonBid ? response.ext.seatnonbid : undefined,
+              atag: emitAtag ? response.ext.prebid.analytics.tags : undefined,
```

Comment:
consistency:

```suggestion
              seatnonbid: response?.ext?.seatnonbid,
              atag: response?.ext?.prebid?.analytics?.tags,
```

If the ternary condition is related to the "OR" condition above, there is perhaps an issue because you can still have access to the values in the `data.response` field.